dataland-frontend/src/components/general/TheClaim.vue

```diff
@@ -0,0 +1,69 @@
+<template>
```

Comment:
Why commented out on NewLandingPage ?   Is this WiP ?

Reply:
This component won't go live on this first version but it will on the next one. I removed it for now

Reply:
The file itself is still in the repo for me. Could you pls re-check if you have really removed it?

Reply:
It is gone now. Closing this conversation.